cmd/cli/kubectl-kyverno/main.go

```diff
@@ -19,6 +19,9 @@ func main() {
 
 func run() error {
 	cmd := commands.RootCommand()
+	if err := cmd.Execute(); err != nil {
+		return fmt.Errorf("Error: %v\n", err)
+	}
```

Comment:
Isn't this a duplicate of https://github.com/kyverno/kyverno/pull/8269/files#diff-85e4f42e8c8553d770a665c82b1c0308f5af6d673ad3b6fbd00234d884ee71b5R28-R30?

Reply:
Yes you are right, changing this.